src/cdf.rs

```diff
@@ -5597,6 +5598,15 @@ pub unsafe fn dav1d_cdf_thread_update(
     }
 }
 
+pub unsafe fn dav1d_cdf_thread_update(
```

Comment:
No, not this one and ones like this.  This one is still `dav1d*` since it uses `Dav1d*` types and it'll become `rav1d*` a few PRs later.  The only `DAV1D_API` functions are in `lib.rs` and the CPU mask setter in `cpu.rs`.

Reply:
Oh actually this one is somewhat unique in that it is a conversation function, but it's private and only called in Rust so it doesn't need `extern "C"` or `#[no_mangle]`.